client/wfnews-war/src/main/angular/src/app/components/report-of-fire/simple-question-page/rof-simple-question-page.component.scss

```diff
@@ -13,7 +13,7 @@
         margin-bottom: 24px;
         border: none;
         border-radius: 8px;
-        height: 60px;
+        height: 80px;
```

Comment:
code conflict here, We may want to use 81px which is same as complex-questions buttons. 
Or use 80px on both.

Reply:
Changed to 81px